paths/node-runner/README.md

```diff
@@ -42,11 +42,40 @@ If you're determined to learn by doing, we recommend using [Node Pilot](https://
 
 ### Hardware
 
-**Hardware Requirements:** 4 CPU’s (or vCPU’s) | 8 GB RAM | 200GB Disk
+**Hardware Requirements:** 4 CPUs (or vCPUs) | 8 GB RAM | 200GB Disk
```

Comment:
Just to be safe: `8 vCPUs | 16 GB RAM | 300 GB Disk`

This will require an update to some sentences below.